scripts/histmakers/mw_with_mu_eta_pt.py

```diff
@@ -438,7 +458,7 @@ def build_graph(df, dataset):
     return results, weightsum
 
 resultdict = narf.build_and_run(datasets, build_graph)
-if smearing_weights:
+if args.smearingWeights:
     wremnants.transport_smearing_weights_to_reco(resultdict)
```

Comment:
Can this go out of the wremnants namespace and just live in muon_calibration (e.g., delete from __init__)?

Reply:
sorry I'm not sure if I understand what you mean, this is defined in `utilities.common.common_parser()` like other arguments.

Reply:
ah I see, done.